step01.md

```diff
@@ -69,6 +69,6 @@ git commit -m 'enter relevant message'
 ## [**next step >>>**](step02.md)
 ---
 ## Keywords
-* [`package.json`](http://browsenpm.org/package.json)
-* npm
-* dependencies
+* [`package.json`](https://docs.npmjs.com/getting-started/using-a-package.json)
+* [`npm`] (https://docs.npmjs.com/getting-started/what-is-npm)
```

Comment:
The space between the `]` and `(` needs to be removed so that the hyperlink renders correctly. Otherwise, it is displayed as below.

![image](https://user-images.githubusercontent.com/10425219/32560869-a58e38ac-c4a2-11e7-8ad5-bc0ef63494e3.png)